src/components/Overlays/OverlayHeaderUsername.tsx

```diff
@@ -13,10 +13,7 @@ export function OverlayHeaderUsername({
 }) {
   return (
     <>
-      <h2
-        className="margin-top-0 margin-bottom-7 text-uppercase text-light underline-base-light text-underline font-sans-xs"
-        style={{ textUnderlineOffset: '0.7em' }}
-      >
+      <h2 className="margin-top-0 margin-bottom-7 text-uppercase text-light underline-base-light text-underline font-sans-xs">
```

Comment:
[blocking question] Does the text underline offset style get added back somewhere?

Reply:
Sorry, I pulled this out as I was experimenting. Can this be pulled into a reusable class as opposed to an inline style?

Reply:
I think so. I doubt we'll have multiple types of text underline offset, so maybe something like:
```
.text-underline-offset { text-underline-offset: 0.7em; }
```